docs/UserGuide.md

```diff
@@ -30,6 +32,23 @@ This section serves to help the user better understand the terminologies used in
 
 # Features
 
+## Input of data: `input`
+
+Stores the data to be processed by modules.
+
+Format: `input <data>`
+
+> :information_source: `data` will be treated as text.
+
+Examples:
+
+- `input HelloWorld!` Enter plain text as data.
```

Comment:
Consider following the same two input examples shown in the example image, in the same order.